### PR TITLE
Added Championify Cask

### PR DESCRIPTION
**Pull request by julian-poidevin**
- Add Championify v2.0.0

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/championify.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'championify' do
+  version '2.0.0'
+  sha256 '092332a6aaf06309180c82a93a1a9b7a874e125cfefe13337db5199899054624'
+
+  url "https://github.com/dustinblackman/Championify/releases/download/#{version}/Championify-OSX-#{version}.dmg"
```
Comment (miccal) [inline Casks/championify.rb]: Can you please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (julian-poidevin) [inline Casks/championify.rb]: Yes, sure. Should be ok now.

**Review by vitorgalvao — APPROVED** [review]

**Comment by miccal** [comment]
Thank you for your contribution.

**Comment by julian-poidevin** [comment]
You're welcome ! This was my first pull request, pretty happy with this 🥇 .

**Pull request merged**